packages/react/src/ReactElementValidator.js

```diff
@@ -214,18 +214,28 @@ function validateChildKeys(node, parentType) {
  *
  * @param {ReactElement} element
  */
+var propTypesMisspellWarningShown;
```

Comment:
Can we move this up to other DEV only variables?

Reply:
Also let's initialise it to false.

Reply:
Added the variable declaration under `if(__DEV__){` which is after import statements. Now working on test.

Reply:
Should the test be added in both `ReactElementValidator-test` and `ReactJSXElementValidator-test`?

Reply:
I think just one is fine.

Reply:
Oh i just added test in both of them, i pushed my code. Should i remove from one ?
Also i have verified all tests were passing and ran build in my local example which was reproducing the error.
I was wondering if i should add another test it(`should not show any errors is correct propTypes property assignment is padded)`?